dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/DefaultRelationshipService.java

```diff
@@ -76,6 +76,10 @@ public Set<RelationshipItem> findRelationshipItems(
           case RELATIONSHIP -> throw new IllegalArgumentException("Unsupported type");
         };
     return relationshipItems.stream()
+        .filter(
+            ri ->
+                ri.getRelationship().getFrom().equals(ri)
```

Comment:
I got bitten by using `.equals` due to our equals implementations in the past 😅 just double-checking that this is safe.

Reply:
it was more related to our many mappings and that the `equals` implementations surprised me in the past 😅 we should be ok here since relationshipItems come from our private store